### PR TITLE
Week3

### PR DESCRIPTION
**Pull request by BogUkraine**

### CODE REVIEW

File: dist/bundle.js [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
...pack_require__(/*! ./../helpers/combineURLs */ \"./node_modules/axios/lib/helpers/combineURLs.js\");\n/**\n * Throws a `Cancel` if cancellation has been requested.\n */\n\n\nfunction throwIfCancellationRequested(config) {\n  if (config.cancelToken) {\n    config.cancelToken.throwIfRequested();\n  }\n}\n/**\n * Dispatch a request to the server using the configured adapter.\n *\n * @param {object} config The config that is to be used for the request\n * @returns {Promise} The Promise to be fulfilled\n */\n\n\nmodule.exports = function dispatchRequest(config) {\n  throwIfCancellationRequested(config); // Support baseURL config\n\n  if (config.baseURL && !isAbsoluteURL(config.url)) {\n    config.url = combineURLs(config.baseURL, config.url);\n  } // Ensure headers exist\n\n\n  config.headers = config.headers || {}; // Transform request data\n\n  config.data = transformData(config.data, config.headers, config.transformRequest); // Flatten headers\n\n  config.headers = utils.merge(config.headers.common || {}, config.headers[config.method] || {}, config.headers || {});\n  utils.forEach(['delete', 'get', 'head', 'post', 'put', 'patch', 'common'], function cleanHeaderConfig(method) {\n    d
```
Comment (EugenMyhed) [inline dist/bundle.js:469]: Add dist folder to gitignore

File: homework_1.html [hunk 2]
```diff
@@ -3,53 +3,28 @@
     <head>
         <meta charset="utf-8">
         <title>Todo list</title>
-        <link rel="stylesheet" href="homework_1.css">
+        <link rel = "stylesheet" href = "./dist/main.css">
     </head>
     <body>
         <div class = "wrapper">
```
Comment (EugenMyhed) [inline homework_1.html:9]: try to use the BEM methodology

File: Todolist/buttons.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const hide_but = document.querySelector(".hide_button");
```
Comment (EugenMyhed) [inline Todolist/buttons.js:1]: Use full names.
For example hideButton/showButton( hide_button/show_button )

Comment (velmyk) [reply Todolist/buttons.js:1]: use camelCase for JavaScript variables

File: Todolist/homework_1.css [hunk 4]
```diff
@@ -61,25 +62,58 @@ h2 {
...
     background: #5f5f5f;
     color: white;
     border-radius: 5px;
 }
+
+.form_button {
+    width: 110px;
+}
+
+.hide_button {
```
Comment (velmyk) [inline Todolist/homework_1.css:79]: Use BEM methodology for classes naming

File: Todolist/index.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        });
+    }
+    console.log(response);
+  })
+  .catch(function (error) {
+    console.log(error);
+  });
+
+window.addEventListener('load', () => {
+    add_button();
```
Comment (velmyk) [inline Todolist/index.js:21]: Name of this 3 methods sound like call to action. If I see this line of code, I think that button should be created.
Better to put your event listeners to some method with name `initEventHandlers` and call in on window load

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (velmyk) [inline .gitignore:1]: add `dist/bundle.js` to gitignore. Generated files should not be in sourse control

File: homework_1.html [hunk 7]
```diff
@@ -3,53 +3,28 @@
     <head>
         <meta charset="utf-8">
         <title>Todo list</title>
-        <link rel="stylesheet" href="homework_1.css">
+        <link rel = "stylesheet" href = "./dist/main.css">
```
Comment (velmyk) [inline homework_1.html:6]: Remove spaces

File: homework_1.html [hunk 8]
```diff
@@ -3,53 +3,28 @@
...
-            <div class = "sidebar_left">
-                <h2>Pure JavaScript</h2>
-                <div class = "input_button">
+             <div class = "header"><h1>TODO List</h1></div>
+             <div class = "sidebar_left">
+                 <h2>Pure JavaScript</h2>
+                 <div class = "input_button">
                     <input type = "text" placeholder = "What to do?" class = "textBox">
-                    <input type = "button" value = "Button" class = "button">
+                    <input type = "button" value = "Button" class = "button form_button">
```
Comment (velmyk) [inline homework_1.html:15]: Remove spacees from html

**Pull request merged**